engine/class_modules/sc_shaman.cpp

```diff
@@ -8017,7 +8017,13 @@ struct flame_shock_t : public shaman_spell_t
   {
     double m = shaman_spell_t::composite_crit_chance();
 
-    m += p()->talent.skybreakers_fiery_demise->effectN( 3 ).percent();
+    if ( !p()->is_ptr() )
```

Comment:
This guard isn't necessary, but is left as a reminder to delete this code after the PTR for 11.0.5 is mainlined.